Framework/Algorithms/src/CreatePSDBleedMask.cpp

```diff
@@ -227,10 +227,10 @@ bool CreatePSDBleedMask::performBleedTest(
   for (; top < topEnd; ++top, ++bot) {
     const int topIndex = tubeIndices[top];
     const int botIndex = tubeIndices[bot];
-    const MantidVec &topY = inputWS->readY(topIndex);
-    const MantidVec &botY = inputWS->readY(botIndex);
-    const MantidVec &topX = inputWS->readX(topIndex);
-    const MantidVec &botX = inputWS->readX(botIndex);
+    auto &topY = inputWS->y(topIndex);
+    auto &botY = inputWS->y(botIndex);
+    auto &topX = inputWS->x(topIndex);
+    auto &botX = inputWS->x(botIndex);
     for (int j = 0; j < numBins; ++j) {
       double topRate(topY[j]), botRate(botY[j]);
       if (isRawCounts) {
```

Comment:
This is doing a conversion from counts to frequencies when necessary. I think you could maybe use `frequencies()` here.